src/cml.js

```diff
@@ -294,7 +276,7 @@ class CML {
     let comment;
     const updatableComment = (comments) => {
       return comments.reverse().find(({ body }) => {
-        return body.includes(watermark);
+        return watermark === null || watermark.isIn(body);
```

Comment:
silly nitpick just in case is `undefined`
```suggestion
        return !watermark || watermark.isIn(body);
```

Reply:
That's a good catch - thank you!

Reply:
Addressed in https://github.com/iterative/cml/pull/1309/commits/84403766e70647da2175ce93af6629d1df322ccc